### PR TITLE
feat: web UI skeleton

### PR DESCRIPTION
**Pull request by mfuhol-weka**

### CODE REVIEW

File: internal/web/disabled.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//go:build !web
+
+package web
+
+import "fmt"
+
+func IsEnabled() bool {
```
Comment (bohdand-weka) [inline internal/web/disabled.go:7]: probably this function should be also in server.go

File: internal/web/server.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+//go:build web
+
+package web
+
+import (
+	"github.com/weka/gohomecli/internal/web/api"
```
Comment (bohdand-weka) [inline internal/web/server.go]: minor: please, sort imports with goimports

File: internal/web/api/k3s.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+type K3sInstallRequest struct {
+	Interface   string   `json:"interface"`
+	Hostname    string   `json:"hostname"`
+	NodeIP      string   `json:"node_ip"`
+	ExternalIPs []string `json:"external_ips"`
+}
+
+func postK3sInstall(w http.ResponseWriter, r *http.Request) {
```
Comment (bohdand-weka) [inline internal/web/api/k3s.go]: got a little bit confusion from naming `postK3sInstall` sounds like hook which runs after k3s installed 😄

**Review by bohdand-weka — APPROVED** [review]
looks good

File: internal/web/disabled.go (hunk 1 above)
Comment (mfuhol-weka) [reply internal/web/disabled.go:7]: thank you for pointing out, forgot to add one

File: internal/web/server.go (hunk 2 above)
Comment (mfuhol-weka) [reply internal/web/server.go]: forgot to setup pre-commit locally, good point

File: internal/web/api/k3s.go (hunk 3 above)
Comment (mfuhol-weka) [reply internal/web/api/k3s.go]: yeah, this one is a bit tricky. I'll rename POST methods just to an action like 'k3sInstall`

**Pull request merged**